### ISSUE TITLE
Sheffield DLP Trainee Outreach - Location for flyers and posters

### ISSUE THREAD

**Issue by MattCFry** [CodeYourFuture/CYF-Staff-Tickets#814]
### Background

We will need to create a list of places where communities meet to make sure we are targeting them for places to hand out flyers and put up posters. 

### Acceptance criteria

- [x] Search for as many relevant places as possible and add them under the 'Community Locations' tab in [Sheffield - Trainee outreach.](https://docs.google.com/spreadsheets/d/1ivm9ioUt9ps4KYbPNnP4hX7uUvlo6juenWLmJMruoR8/edit#gid=0)

### Owner

@mattcfry

### Stakeholder

@amyCYF

### Approver

@amyCYF

**Comment by MattCFry** [comment]
@AmyCYF, who should I attempt to set up the meetings for? G was handing space last time I checked

**Comment by AmyCYF** [comment]
This isn't space, this is a list of places we can leave flyers and posters. e.g Libraries, community centres. It needs some imagination to identify where the people we want to target will see the promotional material.

**Comment by MattCFry** [comment]
@AmyCYF let me know if I should progress with this

**Comment by AmyCYF** [comment]
Are you still waiting on my feedback @MattCFry ? I see this is marked as done?

**Comment by MattCFry** [comment]
Yes, please :) I just closed it to move it along.

**Comment by AmyCYF** [comment]
What do you need me to review / feedback on?

**Comment by MattCFry** [comment]
Is the number of flyer locations I've collected sufficient for the next phase?

**Comment by AmyCYF** [comment]
Yup looks good - next step would be the 'how'. Worth a new ticket on the strategy to get the flyers into these space
